.eslintrc.json

```
@@ -14,7 +14,7 @@
         ],
         "linebreak-style": [
             "error",
-            "unix"
+            "windows"
```

```suggestion
            "unix"
```

It'd be better to keep this change to your local setup